apps/asap-server/src/controllers/events.ts

```diff
@@ -54,6 +54,8 @@ export default class Events implements EventController {
         [],
       );
 
+    filters.push('data/hidden/iv eq false');
```

Comment:
If you use `ne true` we don't need to migrate the old events. Does `eq false` match the `null` as well?

Reply:
damn, i totally didn't think of that, nice one!